libs/@guardian/source-foundations/src/typography/stories/typography.mdx

```diff
@@ -97,4 +97,4 @@ _Note:_ this font is not available on [theguardian.com](https://theguardian.com)
 ### Specimen
 
 <Canvas of={TypographyStories.Presets} />
-import {textEgyptian17} from 'libs/@guardian/source-foundations/vendor/typography/css';
```

Comment:
This looks like an instance of VS Code automatically adding an import, but can be safely removed as it's not meant to be there.

Reply:
the whole import?

Reply:
Yes, the whole import. The `<Canvas>` element _should_ be the last thing in that file.

Reply:
Good timing with my reply as you've literally just done that 😅